<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabPicture">
     <attribute name="icon">
      <iconset resource="ImageClipping.qrc">
       <normaloff>:/picture.png</normaloff>:/picture.png</iconset>
     </attribute>
     <attribute name="title">
      <string>图片</string>
     </attribute>
     <widget class="QLabel" name="darkColorLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="inputDirButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>打开批处理目录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>宽</string>
      </property>
     </widget>
     <widget class="QLabel" name="lightColorLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>149</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>高</string>
      </property>
      <property name="buddy">
       <cstring>heightEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputDirEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="outputDirButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>打开输出目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="darkColorButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>深色背景色</string>
      </property>
     </widget>
     <widget class="QPushButton" name="lightColorButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>浅色背景色</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paddingEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputDirEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="heightEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>边距</string>
      </property>
      <property name="buddy">
       <cstring>paddingEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="widthEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="runButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>GO！</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabText">
     <attribute name="icon">
      <iconset resource="ImageClipping.qrc">
       <normaloff>:/text.png</normaloff>:/text.png</iconset>
     </attribute>
     <attribute name="title">
      <string>文本</string>
     </attribute>
     <widget class="QPushButton" name="inputTextListButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>打开文本列表</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputTextListEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputTextDirEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="outputTextDirButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>打开输出目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runTextButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>GO！</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editRuleButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑规则</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageClipping.qrc"/>
 </resources>
 <connections/>
</ui>
